openslides/motions/static/js/motions/pdf.js

```diff
@@ -657,9 +659,22 @@ angular.module('OpenSlidesApp.motions.pdf', ['OpenSlidesApp.core.pdf'])
                     image_sources = image_sources.concat(tmp_image_sources);
                 });
 
-                //post-request to convert the images. Async.
-                $http.post('/core/encode_media/', JSON.stringify(image_sources)).then(function (success) {
-                    var converter = PdfMakeConverter.createInstance(success.data.images);
+                var image_map = [];
+                _.forEach(image_sources, function (image_source) {
+                    image_map[image_source] = PDFLayout.imageURLtoBase64(image_source);
```

Comment:
I agree, that might not be easy to comprehend at first glance.
we need to get all the images URLs, convert them into promises, asynchronously solve all the promises and again need the information about the original URL. Somehow.

You see why this feature took me so long ;-)